nipype/interfaces/niftyseg/em.py

```diff
@@ -0,0 +1,146 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+
+"""
+Nipype interface for seg_EM.
+
+The em module provides higher-level interfaces to some of the operations
+that can be performed with the seg_em command-line program.
+
+Examples
+--------
+See the docstrings of the individual classes for examples.
+
+Change directory to provide relative paths for doctests
+    >>> import os
+    >>> filepath = os.path.dirname( os.path.realpath( __file__ ) )
+    >>> datadir = os.path.realpath(os.path.join(filepath, '../../testing/data'))
+    >>> os.chdir(datadir)
+"""
+
+from ..base import (TraitedSpec, File, traits, CommandLineInputSpec,
+                    InputMultiPath)
+from .base import NiftySegCommand, get_custom_path
+
+
+class EMInputSpec(CommandLineInputSpec):
+    """Input Spec for EM."""
+    in_file = File(argstr='-in %s',
+                   exists=True,
+                   mandatory=True,
+                   desc='Input image to segment',
+                   position=4)
+
+    mask_file = File(argstr='-mask %s',
+                     exists=True,
+                     desc='Filename of the ROI for label fusion')
+
+    # Priors
+    no_prior = traits.Int(argstr='-nopriors %s',
+                          mandatory=True,
+                          desc='Number of classes to use without prior',
+                          xor=['prior_4D', 'priors'])
+
+    prior_4D = File(argstr='-prior4D %s',
+                    exists=True,
+                    mandatory=True,
+                    desc='4D file containing the priors',
+                    xor=['no_prior', 'priors'])
+
+    desc = 'List of priors filepaths.'
```

Comment:
yes, it was for style and not go over 80 chars when they are very long but I should put this one back there as well as the other one.